lms/templates/fields/field_dropdown_account.underscore

```diff
@@ -20,6 +20,14 @@
         <span class="u-field-value-readonly"></span>
     <% } else { %>
         <select name="select" id="u-field-select-<%- id %>" aria-describedby="u-field-help-message-<%- id %>">
+            <% if (preOptions) { %>
+                <optgroup label="Common Time Zone">
```

Comment:
@kevinjkim Per discussion below, please change this label to "Country Time Zones".